### PR TITLE
Updated <genre> and order for individual instruments

### PR DESCRIPTION
**Pull request by danieljray**
Updated <genre> and order for individual instruments

### CODE REVIEW

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -7867,346 +7844,7 @@
...
...ler ctrl="98" value="7"/></MidiAction><MidiAction name="mixt"><descr>set stop Mixture</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="8"/></MidiAction><MidiAction name="bass18"><descr>set stop Fagott 16</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="9"/></MidiAction><MidiAction name="trom16"><descr>set stop Trombone 16</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="10"/></MidiAction><MidiAction name="bomb32"><descr>set stop Bombarde 32</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="11"/></MidiAction><MidiAction name="trum8"><descr>set stop Trumpet</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="12"/></MidiAction><MidiAction name="CP+1"><descr>set stop P+I</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="13"/></MidiAction><MidiAction name="CP+2"><descr>set stop P+II</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="14"/></MidiAction><MidiAction name="CP+3"><descr>set stop P+III</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="15"/></MidiAction></Channel><genre>common</genre><genre>orchestra</genre></Instrument>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: Something went wrong here

File: share/instruments/instruments.xml [hunk 2]
```diff
@@ -5141,8 +5119,8 @@
                   <genre>marching</genre>
             </Instrument>
             <Instrument id="snare-drum">
-                  <longName>Concert Snare Drum</longName>
-                  <shortName>Con. Sn.</shortName>
+                  <longName>Snare Drum</longName>
+                  <shortName>Sn. Dr.</shortName>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: Same issue with mtests  as @lasconic had to fix last time

File: share/instruments/instruments.xml [hunk 3]
```diff
@@ -5122,7 +5100,7 @@
             <name>Percussion - Unpitched</name>
             <Instrument id="drumset">
                   <longName>Drumset</longName>
-                  <shortName>D. Set</shortName>
+                  <shortName>Drs.</shortName>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: Same issue with mtests  as @lasconic had to fix last time

File: share/instruments/instruments.xml (hunk 1 above)
Comment (danieljray) [reply share/instruments/instruments.xml]: It i just commented out.

Comment (Jojo-Schmitz) [reply share/instruments/instruments.xml]: It was commended out before but now it turned into a one-liner

File: share/instruments/instruments.xml (hunk 2 above)
Comment (Jojo-Schmitz) [reply share/instruments/instruments.xml]: Still an issue left with mtests, see https://travis-ci.org/musescore/MuseScore/jobs/364047450#L5030-L5059, Cornet vs. Trumpet

**Comment by Jojo-Schmitz** [comment]
Still an issue left with mtests, see https://travis-ci.org/musescore/MuseScore/jobs/364047450#L5030-L5059, Cornet vs. Trumpet, sorry missed that in the 1st round

File: share/instruments/instruments.xml (hunk 1 above)
Comment (lasconic) [reply share/instruments/instruments.xml]: Let's delete it.

Comment (Jojo-Schmitz) [reply share/instruments/instruments.xml]: along with all the other disabled reminders of Aeolus?

**Comment by lasconic** [comment]
I squashed the commits and did a new PR here : https://github.com/musescore/MuseScore/pull/3619

@danieljray  please base your future work on the latest version of 2.3 branch when PR https://github.com/musescore/MuseScore/pull/3619 is merged.

**Pull request closed without merging**